extensions/webjars-locator/runtime/src/main/resources/META-INF/quarkus-extension.yaml

```diff
@@ -0,0 +1,11 @@
+---
+name: "WebJar Locator"
+metadata:
+  short-name: "webjars-locator"
+  keywords:
+  - "web"
+  - "webjar"
+  guide: "https://quarkus.io/guides/rest-json"
```

Comment:
This guide should be empty if there is no guide for this, although it would be great if you could add a note about this to the HTTP reference instead.

Reply:
@Christopher-Chianelli How about adding a really short guide for webjars here: https://github.com/quarkusio/quarkus/tree/master/docs/src/main/asciidoc ?

Also, an integration test here to prove it works natively would be great:
https://github.com/quarkusio/quarkus/tree/master/integration-tests